src/main/java/com/google/api/codegen/discovery/config/java/JavaTypeNameGenerator.java

```diff
@@ -61,12 +62,23 @@ public String getSubpackage(boolean isRequest) {
   }
 
   @Override
-  public String formatValue(String value, Field.Kind kind) {
-    switch (kind) {
-      case TYPE_STRING:
-        return "\"" + value + "\"";
-      default:
-        throw new IllegalArgumentException("unsupported kind: " + kind.toString());
+  public String getExample(String format) {
```

Comment:
Right, this function is implemented in each language-specific `TypeNameGenerator`, is that what you mean?

Once we get to Node.js, this whole function will have to be reimplemented under `NodeJSTypeNameGenerator`, this class is Java specific.